module-2/data-analysis-with-pandas/your-code/Pandas project.ipynb

```diff
@@ -9,7 +9,8 @@
     "import pandas as pd\n",
```

Comment:
- You only have around 1.5% of missing values in your height and weight features. In this case, as you have enough data, it's better to drop them instead of fill with mean or whatever other basic statistic. This way has some implications in analysis that must be assess beforehand. """Utilizo la funccion mean_by la cual me devuelve como keys la columna por la q agrupo y su media por la columna q defino, la funcion la cree en un fichero separado y la importe"""
- It doesn't make sense to plot a scatter on a categorical variable, only numeric variables. Medal and it's transformation in number of medals are categorical, even if it has been coded in "numbers". """Midamos si existe alguna relacion evidente entre la altura de los atletas de atletimos y las medallas ganadas"""
- You could have run additional relationship analysis between age and height or/and age and weight. 
- You could also have run some kind of analysis on categorical variables through groupby tables and some relevant bar plots. 
- As you only have around 2000 non missing medal data, I would suggest to remove this variable from your analysis (and change the goal of the analysis) or keep only the 2000 records with medal data in case you want include this feature in your exploratory analysis. Your results on this one are quite biased. 
- One of the first things you should do when you begin an exploratory analysis is to get the description of all of them according to its type (categorical, ordinal, interval....) in order to know what kind of analysis you can run based on data type. This is a good starting point that guides further steps.